File: template.hs
#!/usr/bin/env runghc
import Control.Parallel.Strategies
import Text.Printf
main = getContents >>= mapM_ putStrLn . zipWith caseno [1..] . solve . lines
caseno i r = "Case #"++show i++": "++r
solve (c:cs) = parMap rpar (toString . answer) . take (read c) . analyze $ cs ::[String]
analyze = repeat . const 0.0
toString = printf "%.6f" ::Double->String
answer = id

File: template.py
#!/usr/bin/env python

def solve(i):
    return "%.6f" % i

if __name__=='__main__':
    def analyze(i):
        return 0.0
    xs = map(analyze,range(int(raw_input())))
    from multiprocessing import Pool
    for i,v in enumerate(Pool().imap(solve,xs)):
        print("Case #"+str(i+1)+": "+str(v))
